README.md

```diff
@@ -52,5 +52,6 @@ Please report course issues using the [Issues](https://github.com/DoomHammer/plu
 - https://github.com/devspace/awesome-github-templates
 - https://github.com/stevemao/github-issue-templates
 - https://github.com/ligurio/awesome-ci
+- https://github.com/hockeyhero/nearest
```

Comment:
Hi! I think your link was intended to go under the `List of Awesome Projects` below.

Thanks for adding it and sorry for the late response!